source/cloud-security/amazon/services/supported-services/security-hub.rst

```diff
@@ -0,0 +1,166 @@
+
+    .. thumbnail:: /images/aws/security-hub-policies.png
+      :align: center
+      :width: 70%
+
+When integrating Security Hub and Organizations, there is the option to use a feature called `central configuration <https://docs.aws.amazon.com/securityhub/latest/userguide/central-configuration-intro.html>`_ to set up and manage Security Hub for the organization. It is strongly recommended to use central configuration because it lets the administrator customize security coverage for the organization.
+
+Types of Security Hub integration with EventBridge
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Security Hub events and insights can be stored in S3 buckets by integrating Security Hub with EventBridge as it is carried out, for example, for the :ref:`Amazon WAF integration <amazon_waf>`. 
+The three available types of events are:
+
+- *Security Hub Findings - Imported*: This event type is automatically sent by Security Hub to EventBridge. It includes all new findings and updates to existing findings. Each event contains a single finding.
+- *Security Hub Findings - Custom Action*: This event type is sent by Security Hub to EventBridge when custom actions are triggered. The events are associated with the findings of the custom actions.
+- *Security Hub Insight Results*: This event type is used to process the Security Hub Insights. Custom actions can be used to send sets of insight results to EventBridge. Insight results are the resources that match an insight.
+
+   .. note::
+      To send the last two types of events to EventBridge, you need to create a custom action in Security Hub. Please refer to the Security Hub `documentation <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-custom-actions.html>`_ to achieve this.
+
+Each type of event contains a `specific format <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-event-formats.html>`_ from which the Wazuh integration takes every relevant ``detail`` field and value along with the ``detail-type`` value.
+
+More information on how to configure each type in the `AWS Security Hub related section <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-integration-types.html>`_.
+
+Amazon Simple Queue Service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Set up a *Standard* type SQS Queue with the default configurations.  You can apply an Access Policy similar to the following example, where ``<region>``, ``<account-id>``, and ``<s3-bucket>`` are the region, account ID, and the name you are going to provide to the S3 bucket.
```

Comment:
Why? What for?

Reply:
This is related to https://github.com/wazuh/wazuh-documentation/pull/7111#discussion_r1535601550